app/controllers/v1/sessions_controller.rb

```diff
@@ -58,10 +62,11 @@ def new
 
         redirect_to url.to_s
       else
-        render_login(type)
+        render_login(type, operation)
```

Comment:
I don't think we need `operation` rendered. @asg5704 you don't need operation reflected back to the FE right?

Reply:
@bosawt that is correct we do not need the operation back. The `operation` is just used for logging in the API.